buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -306,6 +309,32 @@ class ClusterFormationTasks {
         }
     }
 
+    /** Adds a task to create keystore */
+    static Task configureCreateKeyStoreTask(String name, Project project, Task setup, NodeInfo node) {
+        File esKeyStoreUtil = Paths.get(node.homeDir.toString(), "bin/" + "elasticsearch-keystore").toFile()
+        Task createKeyStore = project.tasks.create(name: name, type: LoggedExec, dependsOn: setup) {
```

Comment:
One more thing: you should use `configureExecTask` so this will work correctly on windows.